geo_activity_playground/webui/blueprints/eddington_blueprints.py

```diff
@@ -0,0 +1,266 @@
+import datetime
+from math import ceil
+
+import altair as alt
+import numpy as np
+import pandas as pd
+from flask import Blueprint
+from flask import render_template
+from flask import Request
+from flask import request
+
+from ...core.activities import ActivityRepository
+from ...core.meta_search import apply_search_query
+from ..columns import column_distance
+from ..columns import column_elevation_gain
+from ..search_util import search_query_from_form
+from ..search_util import SearchQueryHistory
+
+
+def register_distance_eddington_blueprint(
+    repository: ActivityRepository, search_query_history: SearchQueryHistory
+) -> Blueprint:
+    blueprint = Blueprint(f"distance_eddington", __name__, template_folder="templates")
+
+    @blueprint.route("/")
+    def index():
+        return _render_eddington_template(
+            repository,
+            request,
+            search_query_history,
+            "distance",
+            column_distance,
+        )
+
+    return blueprint
+
+
+def register_elevation_eddington_blueprint(
```

Comment:
I would like to have one blueprint per file. We can make the Eddington stuff one blueprint with two routes, I think that would make sense.